### PR TITLE
Fe error pages

### PR DESCRIPTION
**Pull request by Searay-330**
#190 fixed

### CODE REVIEW

File: client/routes.js [hunk 1]
```diff
@@ -40,6 +41,7 @@ export default (
         <Route path="/courses/:course/assignments/:assignment" component={Assignment} />
       </Route>
     </Route>
+    <Route path="*" component={ErrorPages} error="404" />
```
Comment (theknickerbocker) [inline client/routes.js:44]: Could you explain why you have to set error=404? I thought the reason for having a single errorpages component was that the errorpages would switch based on the actual error status

Comment (Searay-330) [reply client/routes.js:44]: so if you used this inside a api call then you would but this will be a 404 no matter what so i hard coded it.

Comment (theknickerbocker) [reply client/routes.js:44]: fair

**Review by theknickerbocker — APPROVED** [review]
Buenos noches señor

**Pull request merged**